component/libfuse/libfuse.go

```diff
@@ -220,7 +220,7 @@ func (lf *Libfuse) Configure(_ bool) error {
 	log.Trace("Libfuse::Configure : %s", lf.Name())
 
 	// >> If you do not need any config parameters remove below code and return nil
-	conf := LibfuseOptions{}
+	conf := LibfuseOptions{IgnoreOpenFlags: true}
```

Comment:
Why are we setting to true here? Will this cause the customers decision to be overridden?

Reply:
The default value of the `ignore-open-flags` will be `true` when it is not specified in the config as well as the CLI.
It is set to `false` when the flag is either specified as false in the config or the CLI. I have tested the combinations and it is working as expected. I will try to add some test cases for the different combinations around this.

Reply:
Added relevant test cases.